README.md

```diff
@@ -14,26 +14,28 @@ If there is any information that would be helpful but is missing from the change
-| Flatcar Linux           | stable  | Rolling release | N/A              | TBD                                                               | Coming soon                 |
-| Nixos                   | 21.11   | Dec, 2021       | TBD              | TBD                                                               |                             |
-| RedHat Enterprise Linux | 7       | N/A             | Aug, 2021        | [x86_64](rhel/x86_64/7.md)                                        |                             |
-| RedHat Enterprise Linux | 8       | May, 2024       | N/A              | [x86_64](rhel/x86_64/8.md)                                        |                             |
-| Rocky Linux             | 8       | May, 2024       | Feb, 2024        | [x86_64](rocky/x86_64/8.md) [arm64](rocky/aarch64/8.md)           |                             |
-| Talos                   | 1.2.3   | Sep, 2026       | Sep, 2026        | [Changelog](talos/v1.md)                                          |                             |
-| Ubuntu                  | 18.04   | Apr, 2028       | Jan, 2028        | [x86_64](ubuntu/x86_64/18_04.md) [arm64](ubuntu/aarch64/18_04.md) |                             |
-| Ubuntu                  | 20.04   | Apr, 2030       | Jan, 2030        | [x86_64](ubuntu/x86_64/20_04.md) [arm64](ubuntu/aarch64/20_04.md) |                             |
-| Ubuntu                  | 22.04   | Apr, 2032       | Jan, 2032        | [x86_64](ubuntu/x86_64/22_04.md) [arm64](ubuntu/aarch64/22_04.md) |                             |
-| VMware ESXi             | 6.7     | Oct, 2022       | Aug, 2022        | N/A                                                               | Not available on latest HW. |
-| VMware ESXi             | 7.0     | Apr, 2025       | Jan, 2025        | TBD                                                               |                             |
-| VyOS                    | TBD     | ???             | ???              | TBD                                                               | Coming soon                 |
-| Windows Server          | 2019    | Jan, 2024       | Sept, 2023       | [x86_64](windows/x86_64/2019.md)                                  |                             |
-| Windows Server          | 2022    | Oct, 2026       |                  | [x86_64](windows/x86_64/2022.md)                                  |                             |
-
+| OS                      | Version | Upstream EOL  | EM Estimated EOL |  Changelog                      | Notes |
+| ---                     | ---   | ---             | ---        | ---                                                               | --- |
+| Ubuntu                  | 18.04 | Apr, 2028       | Jan, 2028  | [x86_64](ubuntu/x86_64/18_04.md) [arm64](ubuntu/aarch64/18_04.md) | |
```

Comment:
Is there a reason why Ubuntu, Debian, Alma, Rocky, an VMWare are in chronological order, but RHEL, CentOS, and Windows are reversed?

Reply:
No, need to talk to someone that consumes this page about which one makes more sense. They were simply written at different times.